src/vue-components/user-auth.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+    <div>
+        <button v-show="!isLoggedIn" class="uk-button uk-button-primary" @click="logIn">
+            Log in
+        </button>
+        <button v-show="isLoggedIn" class="uk-button uk-button-primary" @click="logOut">
+            Log out
+        </button>
+        <div v-show="logInProgress" class="alpheios-user-auth__message-box">
+            Please be patient while we are logging you in ...
```

Comment:
May be it is better to place it to l10n.messages?

Reply:
Agree, will add this. Was not using it because this is an intermediary solution, not the final one, but it's good to have it right from the start.